builtin/logical/pki/backend.go

```diff
@@ -748,3 +759,135 @@ func (b *backend) decrementTotalRevokedCertificatesCountNoReport() uint32 {
 	newRevokedCertCount := atomic.AddUint32(b.revokedCertCount, ^uint32(0))
 	return newRevokedCertCount
 }
+
+// runUnifiedTransfer meant to run as a background, this will process all and
+// send all missing local revocation entries to the unified space if the feature
+// is enabled.
+func (b *backend) runUnifiedTransfer(sc *storageContext) {
+	isPerfStandby := b.System().ReplicationState().HasState(consts.ReplicationDRSecondary | consts.ReplicationPerformanceStandby)
+
+	if isPerfStandby || b.System().LocalMount() {
+		// We only do this on active enterprise nodes, when we aren't a local mount
+		return
+	}
+	config, err := b.crlBuilder.getConfigWithUpdate(sc)
+	if err != nil {
+		b.Logger().Error("failed to retrieve crl config from storage for unified transfer background process",
+			"error", err)
+		return
+	}
+
+	if !config.UnifiedCRL {
+		// Feature is disabled, no need to run
+		return
+	}
+
+	clusterId, err := b.System().ClusterID(sc.Context)
+	if err != nil {
+		b.Logger().Error("failed to fetch cluster id for unified transfer background process",
+			"error", err)
+		return
+	}
+
+	if clusterId == "" {
+		b.Logger().Error("got a blank cluster id for unified transfer background process")
```

Comment:
This shouldn't happen FWIW, I err if this is blank.

Reply:
Your right, I could have sworn that check wasn't there when I last looked, will remove.

Reply:
Fixed.